dockers/docker-platform-monitor/lm-sensors.sh

```diff
@@ -18,6 +18,3 @@ fi
 
 /usr/bin/sensors > /dev/null 2>&1
 
-# Currently, there is no way to run sensord in the foreground, so we
-# can't use supervisord. Instead, we just start the service for now.
-service sensord start
```

Comment:
Why is this no longer needed?

Reply:
Sensord is not removed in debian stretch. Can find the detail in this Webpage:https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=865042

Reply:
if this is not available any more what will be the be the sensor daemon in SONiC? do we reduced functionality? in this case we must find a solution before we take it our

Reply:
There would be no sensor daemon after upgrade to stretch as it has been removed from Debian stretch package because of the buggy.  As the maintainer mentioned on this page https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=865042, it is hard to be reintroduced into stretch, so if we need to upgrade pmon container to stretch, we need to find the sensord's deb package and all the packages it depends on. Put all those .deb package in the sonic-buildiamge repo and install it manually rather than using apt-get. Is this solution acceptable?

Reply:
Add jessie to the deb source in sources.list. Then the sensord could work fine.

Reply:
Could doing so potentially result in downloading outdated versions of other packages?

Reply:
adding jessie back is strange, the right approach is the modify the debian packaging file in stretch to build sensord package.